javascript/src/musica_wrapper.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <string>
+#include <memory>
+
+namespace musica_addon {
+
+/**
+ * @brief C++ wrapper for MUSICA functionality
+ */
+class MusicaWrapper {
```

Comment:
Could we be more specific with the descriptions? Since MUSICA functions are written in C++, it could be confusing.
```suggestion
/// @brief C++ wrapper for exposing MUSICA functionality to Node.js
class MusicaWrapper {
```